spring-scenario/config/expectedData.yaml

```diff
@@ -3,15 +3,45 @@ registryItems:
   - {restapi-scenario: nq 0}
   - {spring3-scenario: nq 0}
   - {resttemplate-scenario: nq 0}
+  - {inherit-scenario: nq 0}
   instances:
   - {restapi-scenario: 1}
   - {spring3-scenario: 1}
   - {resttemplate-scenario: 1}
+  - {inherit-scenario: 1}
   operationNames:
   - restapi-scenario: [/create/, '/update/{id}', '/delete/{id}', '/get/{id}']
   - spring3-scenario: [/spring3]
-  - resttemplate-scenario: [/spring-projectc/spring3/, /resttemplate-case/case/resttemplate, /restapi-case/create/, /restapi-case/update/1, /restapi-case/delete/1, /restapi-case/get/1]
+  - resttemplate-scenario: [/spring-projectc/spring3/, /resttemplate-case/case/resttemplate, /restapi-case/create/, /restapi-case/update/1, /restapi-case/delete/1, /restapi-case/get/1, /inherit-case/inherit/child/test]
+  - inherit-scenario: [/inherit/child/test]
 segmentItems:
+- applicationCode: inherit-scenario
+  segmentSize: 1
+  segments:
+  - segmentId: not null
+    spans:
+    - operationName: /inherit/child/test
+      operationId: 0
+      parentSpanId: -1
+      spanId: 0
+      spanLayer: Http
+      startTime: nq 0
+      endTime: nq 0
+      componentId: 14
+      componentName: ''
+      isError: false
+      spanType: Entry
+      peer: ''
+      peerId: 0
+      tags:
+      - {key: url, value: 'http://inherit-scenario:8080/inherit-case/inherit/child/test'}
+      - {key: http.method, value: GET}
+      refs:
+      - {parentServiceId: 0, parentServiceName: /resttemplate-case/case/resttemplate,
+         networkAddressId: 0, entryServiceId: 0, refType: CrossProcess, parentSpanId: 6,
+         parentTraceSegmentId: not null, parentApplicationInstanceId: nq 0,
```

Comment:
I suggest you should assert the `parentTraceSegmentId`.  the format of parentTraceSegmentId looks like this: `${Application[index]}`. Here is the  [doc](https://github.com/SkywalkingTest/skywalking-agent-testcases/blob/master/docs/how-to-write-a-plugin-testcase-cn.md#%E6%9C%9F%E6%9C%9B%E6%95%B0%E6%8D%AE%E6%96%87%E4%BB%B6)